Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,27 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            int tWeight = 50;
+            float tHeight = 1.565F;
+            float tBmi = tWeight / tHeight;
+            int tAge = 19;
+
+            int cWeight = 68;
+            float cHeight = 1.55F;
+            float cBmi = cWeight / cHeight;
+            int cAge = 20;
+
+            int bWeight = 61;
+            float bHeight = 1.41F;
+            float bBmi = bWeight / bHeight;
+            int bAge = 32;
```

Comment:
Looking good so far! You're still needing to read in values from the console using `Console.ReadLine()`. 

    // getting a string
    Console.WriteLine("Enter a string.");
    string value = Console.ReadLine();

There's still a bit to do for numeric variables. Take a look at the [Finding missing pieces (Console)](https://github.com/csinn/CSharp-From-Zero-To-Hero/tree/Chapter1/Homework/1And2#finding-missing-pieces-console) section for ideas.